### ISSUE TITLE
History plugin / get_account not maintaining "last_unstake_time" and "unstaking"

### ISSUE THREAD

**Issue by ScottSallinen** [EOSIO/eos#4114]
I have been trying to use get_account to find out the number of tokens a user is unstaking. 

This is an important metric to show users for our Greymass light wallet, as people are confused that their tokens "dissapear" after they unstake them. (only to see them appear liquid 3 days later).

I have tried querying the API nodes hosted by BPs for accounts I know have unstaked, and I only ever recieve the same results:

"last_unstake_time" :"1970-01-01T00:00:00"
"unstaking":"0.0000 EOS"

On my own api node I have even tried with filter-on = *, to no avail.

Either there is something that *everyone hosting an API node* is missing to properly populate these values, or there is a bug in the collection of this value.

Note: I have tried this with an API node running 1.0.3 as well, still no luck.

**Comment by arhag** [comment]
The fields `deferred_trx_id`, `last_unstake_time`, and `unstaking` in the `voters` table of the system contract are not currently being used by the system contract. In fact, these should probably be removed since they aren't needed anymore, but we first need to make sure that changing the database schema of the voters table on a live network does not break anything. But you certainly cannot rely on these fields containing any useful information.

The information useful for unstaking is stored in the `refunds` table. I recently made a change to chain_plugin and cleos (which will be in the v1.0.4 release tomorrow) that captures the relevant staking information. See PR #4063. If you want to get a head start on using this information in your wallet, I recommend compiling the latest master and trying out that API node.

**Comment by ScottSallinen** [comment]
Thanks @arhag ,
I have compiled the latest master and connected it to the community mainnet. (note: I had to change some occurrences of `self_delegated_bandwidth` to `delegated_bandwidth` in order to have cleos function properly. This is probably related to the previous bug with this part of the struct.)

I `--replay`'d my full node with these relevant test settings:
```
plugin = eosio::wallet_plugin
plugin = eosio::wallet_api_plugin
plugin = eosio::history_plugin
plugin = eosio::history_api_plugin
plugin = eosio::chain_plugin
plugin = eosio::chain_api_plugin
plugin = eosio::http_plugin
plugin = eosio::wallet_plugin
filter-on = *
```

However, when I use cleos, I still do not see anything regarding refunds/unstaking information. An example below (an account I noted to have unstaked 50 EOS recently, http://eosflare.io/account/gm3dmmjqgene):

```
$./cleos get account gm3dmmjqgene
permissions:
     owner     1:    1 EOS546KNQVF1hN9292SF7umg3eKcqwyJfjgN2fYJJSiWAAAHr5CyM
        active     1:    1 EOS546KNQVF1hN9292SF7umg3eKcqwyJfjgN2fYJJSiWAAAHr5CyM
memory:
     quota:     7.953 Kb     used:     4.506 Kb

net bandwidth:
     staked:         18.3500 EOS           (total stake delegated from account to self)
     delegated:       0.0000 EOS           (total staked delegated to account from others)
     used:               250 bytes
     available:        6.904 Mb
     limit:            6.905 Mb

cpu bandwidth:
     staked:         18.3500 EOS           (total stake delegated from account to self)
     delegated:       0.0000 EOS           (total staked delegated to account from others)
     used:             3.096 ms
     available:        1.378 sec
     limit:            1.381 sec

producers:
     eosecoeoseco
```

**Comment by arhag** [comment]
Is this using the cleos compiled with the latest master? What does `cleos version client` return?

Also what is the `server_version` from the output of `cleos get info` and what is the output of `cleos --print-response get account gm3dmmjqgene`?

**Comment by ScottSallinen** [comment]
Apologies -- I compiled again with a fresh repo (must have missed a rebase or something) and it does work!
The unstaking values to properly appear in `cleos` in the latest master.

```
unstaking tokens:
     time of unstake request:  2018-06-14T21:30:46 (funds will be available in 50.83 hr)
     from net bandwidth:             25.0000 EOS
     from cpu bandwidth:             25.0000 EOS
     total:                          50.0000 EOS
```

Feel free to close this issue if you like, or leave it open so that others can see that it will be addressed in 1.0.4.

**Comment by noprom** [comment]
Same problem here, solved by upgrading to 1.0.4.
